templates/zerver/emails/digest.html

```diff
@@ -10,9 +10,9 @@ <h2>Missed private messages</h2>
     <p>You have some missed private messages. Here are some of them:</p>
     <div id='private-messages' style="width: 600px;font-size: 12px;font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;overflow-y: scroll;">
         {% for recipient_block in unread_pms %}
-        <div class='recipient_block' style="background-color: #f0f4f5;border: 1px solid black;margin-bottom: 4px;">
-            <div class='recipient_header' style="color: #ffffff;background-color: #444444;border-bottom: 1px solid black;font-weight: bold;padding: 2px;">{{ recipient_block.header.html|safe }}</div>
-            <div class='message_content' style="background-color: #f0f4f5;margin-left: 1px;margin-right: 2px;">
+        <div class='recipient_block' style="background-color: hsl(192, 20%, 95%);border: 1px solid black;margin-bottom: 4px;">
```

Comment:
https://github.com/zulip/zulip/commit/95d86c8aebc#diff-8d20fd9ec56db73bd51a4ec458fe99b9R617

Changes it as `background-color: hsl(192, 19%, 95%);` and we still have 2 occurrences of these in code base.